scripts/deploy/troubleshooting/get-all-logs

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+function usage() {
+  echo "Usage: $0 <namespace> [since]"
+  echo
+  echo "This script saves logs from all pods within the specified namespace into local files and then creates a tar.gz archive of these logs."
+  echo
+  echo "Arguments:"
+  echo "  namespace   - The namespace from which to retrieve and archive pod logs."
+  echo "  since       - (Optional) Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs."
+  echo
+  echo "Example: $0 mynamespace 24h"
+  echo
+  echo "Note: Logs may contain Personally Identifiable Information (PII). Handle and store them securely."
+  exit 1
+}qq
```

Comment:
calling this now I get an error instead of the `usage()` output:

```
➜  deploy git:(cht-deploy-fixes) ✗ ./troubleshooting/get-all-logs 
./troubleshooting/get-all-logs: line 55: syntax error: unexpected end of file
```

Aha! There's two errant `qq`s that need to be removed:

```suggestion
}
```